enterprise/server/byte_stream_server_proxy/byte_stream_server_proxy.go

```diff
@@ -65,19 +66,19 @@ func (s *ByteStreamServerProxy) Read(req *bspb.ReadRequest, stream bspb.ByteStre
 	defer spn.End()
 
 	if authutil.EncryptionEnabled(ctx, s.authenticator) {
-		metrics.ByteStreamProxyReads.With(
-			prometheus.Labels{metrics.CacheHitMissStatus: "remote-only"}).Inc()
-		return s.readRemote(req, stream)
+		err := s.readRemote(req, stream)
+		recordReadMetrics(req, "uncacheable", err)
```

Comment:
I think you probably want to define these strings "hit", "miss", etc, in metrics and reference them here and below?

Reply:
sg, done